src/Cake.Core/Reflection/AssemblyLoader.cs

```diff
@@ -17,18 +18,21 @@ public sealed class AssemblyLoader : IAssemblyLoader
         private readonly ICakeEnvironment _environment;
         private readonly IFileSystem _fileSystem;
         private readonly IAssemblyVerifier _verifier;
+        private readonly ICakeLog _log;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AssemblyLoader"/> class.
         /// </summary>
         /// <param name="environment">The environment.</param>
         /// <param name="fileSystem">The file system.</param>
         /// <param name="verifier">The assembly verifier.</param>
-        public AssemblyLoader(ICakeEnvironment environment, IFileSystem fileSystem, IAssemblyVerifier verifier)
+        /// <param name="log">The cake log.</param>
+        public AssemblyLoader(ICakeEnvironment environment, IFileSystem fileSystem, IAssemblyVerifier verifier, ICakeLog log)
```

Comment:
Should we regard this as a breaking change?

Reply:
Don't think so in this case,  interface untouched,  created through IoC, only used before addins are initiated and Cake.Tool

Reply:
Sounds good to me!